### PR TITLE
Add link to the future index section in operations manual

### PR DESCRIPTION
**Pull request by Hunterness**
Add link in the future index section in cypher manual.

~~This PR adds the section in question: https://github.com/neo-technology/neo4j-manual-modeling/pull/2448~~

### CODE REVIEW

File: cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc [hunk 1]
```diff
@@ -420,8 +420,7 @@ include::../indexes/drop-a-non-existing-index.asciidoc[leveloffset=+1]
 Two new types of indexes, point and range indexes, will be introduced in a future release.
 They cannot be used in queries yet, but they can be created and dropped for migration purposes.
 These new index types together with text indexes will replace the current b-tree indexes.
-For more details on these new types, see the operations manual.
-// TODO: add the link to the relevant operations manual section when that section exists
+For more details on these new types, see the <<operations-manual#future-indexes, operations manual>>.
```
Comment (Hunterness) [inline cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc]: Or do we want:

```suggestion
For more details on these new types, see the <<operations-manual#future-indexes, Operations manual -> Future indexes>>.
```

**Comment by Hunterness** [comment]
I have not built and checked that the link works so that would be much appreciated if it could be confirmed :)

File: cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc (hunk 1 above)
Comment (AlexicaWright) [reply cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc]: It should be 
```suggestion
For more details on these new types, see the <<operations-manual#future-indexes, Operations Manual -> Future indexes>>.
```

**Review by AlexicaWright — COMMENTED** [review]
One suggestion

File: cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc (hunk 1 above)
Comment (Hunterness) [reply cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc]: That looks like what I wrote :P

I guess it is the capital M that differs in our suggestirons :P

Comment (AlexicaWright) [reply cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc]: Yes, capital M is the difference. LGTM now! 👍

Comment (Hunterness) [reply cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc]: Have you tested that the link works as well?

**Review by AlexicaWright — APPROVED** [review]
The link will work.

**Pull request merged**